packages/next/client/polyfills.js

```diff
@@ -0,0 +1,3 @@
+// Object.assign() is commonly used with React.
+// It will use the native implementation if it's present and isn't buggy.
+Object.assign = require('object-assign')
```

Comment:
Do we need to load `object-assign` at all since the [native support is pretty good](http://kangax.github.io/compat-table/es6/#Object_static_methods_Object.assign) especially when we target only type=module browsers